src/components/axis/axis-render.ts

```diff
@@ -80,15 +80,21 @@ export default class AxisRender extends UIController<AxisOption> {
       ? axisBottom
       : axisLeft;
 
-    const maxTicks =
-      this.option.minStep != null
-        ? Math.min(
-            Math.ceil(Math.max(...ySeriesValue) / this.option.minStep) || 1,
-            this.getMaxTicks(),
-          )
-        : this.getMaxTicks();
-
-    let axis = d3Axis(scale as AxisScale<any>).ticks(maxTicks);
+    let ticks = this.option.ticks;
+    if (!ticks) {
+      ticks =
+        this.option.minStep != null
+          ? Math.min(
+              Math.ceil(Math.max(...ySeriesValue) / this.option.minStep) || 1,
+              this.getMaxTicks(),
+            )
+          : this.getMaxTicks();
+    }
+    if (!Array.isArray(ticks)) {
+      ticks = [ticks];
+    }
+    // @ts-ignore
+    let axis = d3Axis(scale as AxisScale<any>).ticks(...ticks);
```

Comment:
```suggestion
    let axis = d3Axis(scale as AxisScale<any>)
      // @ts-expect-error
      .ticks(...ticks);
```

用 `@ts-expect-error` 更好些？